libs/ballot-interpreter-nh-next/src/interpret.rs

```diff
@@ -336,10 +358,12 @@ pub fn interpret_ballot_card(side_a_path: &Path, side_b_path: &Path, options: &O
         front: InterpretedBallotPage {
             grid: front_grid,
             marks: front_scored_oval_marks,
+            normalized_image: make_normalized_image_buffer(front_image),
```

Comment:
I'm not sure how long this takes, but it's possible it'd be worth parallelizing with `rayon::join`.

Reply:
Cool, will let you investigate when you perf test?